src/Source/C3DTilesGoogleSource.js

```diff
@@ -0,0 +1,59 @@
+import Fetcher from 'Provider/Fetcher';
+import C3DTilesSource from './C3DTilesSource';
+
+/**
+ * @classdesc
+ * An object defining the source connection to a 3DTiles asset of a [Google api](https://tile.googleapis.com).
```

Comment:
`of a [Google..` -> `from a [Google..` ?

Reply:
ok